hedgehog/src/Hedgehog/Internal/Report.hs

```diff
@@ -11,9 +11,9 @@ module Hedgehog.Internal.Report (
   , FailureReport(..)
   , FailedInput(..)
 
-  , ShrinkCount
-  , TestCount
-  , DiscardCount
+  , ShrinkCount(..)
+  , TestCount(..)
+  , DiscardCount(..)
```

Comment:
These are all `Num` + `Integral` already, so you don't need the constructor exposed to get at them if you want it as an `Int` (just use `fromIntegral`)